### ISSUE TITLE
Find a Store: "No store found" message is not vocalized

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#13735]
**Describe the bug**
- If a screen reader user triggers a finds no stores, the message that "no stores were found" is not vocalized by the screen reader, therefore the user doesn't have any feedback about what the result of the search is.

**Tell us the version of Spartacus**
- Library version: 4.0.0

**To Reproduce**
1. Open any Screen Reader (VO, NVDA, JAWS).
2. Go to the Find a Store page
3. Enter an incorrect postal code
4. See the message.
5. Hear the absence of vocalization.

**Expected behavior**
- Screen reader vocalizes the "No stores found in databases" message.
- Add `aria-live="assertive" aria-atomic="true"` to the `<span class="cx-no-stores">` containing the message.

**Screenshots**

![Screen Shot 2021-09-15 at 1 22 17 PM](https://user-images.githubusercontent.com/15113219/133485099-05b2e126-cee7-41e8-b3a9-99fa52ffc947.png)

**Comment by KateChuen** [comment]
Tested on develop branch of spartacus.

![NoStores](https://user-images.githubusercontent.com/45409515/144303669-fef81fa7-7e8b-4705-94ae-7e515888fc2c.png)

![NoStores_1](https://user-images.githubusercontent.com/45409515/144303670-9da8bd9d-ac4d-4193-ae71-198a68673c43.png)


More extensive testing (on other screenreaders and devices) will be done with RC.
